src/hooks/resize-listener.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect, Ref } from "preact/hooks";
+
+function useResizeListener(ref: Ref<HTMLElement | null>, listener: () => void): void {
```

Comment:
The `ref` argument is unused. Can we drop it?

Reply:
Whoops, good catch! will make another PR to remove

Reply:
https://github.com/18F/identity-reporting/pull/35